src/GameLogic/PlayerActions/Craftings/DevilSquareTicketCrafting.cs

```diff
@@ -35,6 +35,6 @@ protected override int GetPrice(int eventLevel)
     /// <inheritdoc />
     protected override byte GetSuccessRate(int eventLevel)
     {
-        return (byte)(80 - (eventLevel * 5));
+        return (byte)(eventLevel < 5 ? 80 : 70); // Future to-do: There is a +10% increase if Crywolf event is beaten
```

Comment:
This is something that appears to have changed throughout the versions. Anyway, here is what the client says:
![image](https://github.com/user-attachments/assets/ae1fd95e-572e-4544-a237-772273c96781)